tools/experiments/README.md

```diff
@@ -20,12 +20,25 @@ For content served from any other domain, you can toggle experiments in the Chro
 AMP.toggleExperiment('experiment')
 ```
 
+## Document opts in an experiment
+Document can choose to opt in a certain experiments. To do that, simply put a meta tag of name `amp-experiments-opt-in` in the head of the HTML document. Its `content` value is a comma separated string of experiment IDs to opt in.
+```HTML
+<head>
+  ...
+  <meta name="amp-experiments-opt-in" content="experiment-a,experiment-b">
+  ...
+</head>
+```
+By doing so, the specified experiments will be enabled for all visitors of the document.
+Note not all experiments allow document level opt-in. For a full list of whitelisted experiments, please see `allow-doc-opt-in` attribute in the project's `prod-config.json` file.  
```

Comment:
Discuss when this should be done. It seems the actual use cases for this are pretty rare.